Interface/src/components/DeviceRow.vue

```diff
@@ -57,13 +57,19 @@
       </select>
     </td>
     <td class="column-four">
+      <input
```

Comment:
Could you add a label here when in mobile view? 
The table headers don't show here, so it might be a bit confusing.